spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/SpatialDropout3D.scala

```diff
@@ -24,16 +24,24 @@ import com.intel.analytics.bigdl.utils.Shape
 import scala.reflect.ClassTag
 
 class SpatialDropout3D[T: ClassTag](val p: Double = 0.5,
-                                    val dim_ordering: DataFormat = DataFormat.NCHW,
+                                    val format: String = "CHANNEL_FIRST",
                                     var inputShape: Shape = null
                                    )(implicit ev: TensorNumeric[T])
   extends KerasLayer[Tensor[T], Tensor[T], T](KerasLayer.addBatch(inputShape)) {
 
   override def doBuild(inputShape: Shape): AbstractModule[Tensor[T], Tensor[T], T] = {
-    val layer = com.intel.analytics.bigdl.nn.SpatialDropout3D(
+
+    var layer = com.intel.analytics.bigdl.nn.SpatialDropout3D(
```

Comment:
Please use `if else` or `case` syntax on `format` instead of the whole layer.....